crates/solidity/inputs/language/src/definition.rs

```diff
@@ -6886,15 +6877,20 @@ codegen_language_macros::compile!(Language(
         ),
         BuiltInType(
             name = "$bytes",
```

Comment:
nit: `$Bytes`?

Reply:
Yep, that's a leftover. I even switched to `%Array` but didn't notice this one. Will change it.

Reply:
Hmm... on second thought, this is different from `%Array` and more similar to `%address` which remains in all lowercase. The rationale behind it is the rules that push the symbol types build them directly from the keywords. But maybe we should change those as well...

Reply:
Changing the other elementary types to produce title case variants is not trivial. There is no usable string transformation function in the graph builder. We could implement one, but I'm not sure it's worth it.

Reply:
>IIUC, that can also be used to bind to an Address type instead of %address.

This would not be possible anyway, cause we could be conflicting with a user-defined `Address` struct/contract/whatever. We still need to prefix built-in types with `%` to avoid conflicts with user definitions.